### PR TITLE
Added Simple vue stuff

### PR DESCRIPTION
**Pull request by GiantRobato**

### CODE REVIEW

**Review by lottiely — APPROVED** [review]
Good

File: client/build/build.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict'
```
Comment (ericpak) [inline client/build/build.js:1]: The build folder shouldn't be included in source control. Add this folder to gitignore please.

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    "semver": "^5.3.0",
+    "shelljs": "^0.7.6",
+    "uglifyjs-webpack-plugin": "^1.1.1",
+    "url-loader": "^0.5.8",
+    "vue-loader": "^13.3.0",
+    "vue-style-loader": "^3.0.1",
+    "vue-template-compiler": "^2.5.2",
+    "webpack": "^3.6.0",
+    "webpack-bundle-analyzer": "^2.9.0",
+    "webpack-dev-server": "^2.9.1",
```
Comment (ericpak) [inline client/package.json:63]: Security errors on this dependency. Look into updating or replacing it.

File: client/src/components/HelloWorld.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
+<template>
```
Comment (ericpak) [inline client/src/components/HelloWorld.vue:1]: delete this please

File: client/src/components/fortune.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-10">
+        <h1>
+            <img src="../assets/fortune-cookie.png" style="display:inline;" alt="fortune cookie" height="42" width="42">
```
Comment (ericpak) [inline client/src/components/fortune.vue:6]: Please no inline styling. Not referring to display:inline but to styling within the tag. Let's figure out if we want to have internal styling or external styling next Tuesday.

File: client/src/components/fortune.vue [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        <hr><br><br>
+        <button type="button" class="btn btn-success btn-sm">Add Fortune</button>
+        <br><br>
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <th scope="col" style="width: 50%">Fortune</th>
+              <th scope="col" style="width: 20%">Author</th>
+              <th scope="col" style="width: 10%">Reviewed?</th>
+              <th style="width: 20%"></th>
```
Comment (ericpak) [inline client/src/components/fortune.vue:18]: No inline styling please.

File: client/src/components/HelloWorld.vue (hunk 3 above)
Comment (ericpak) [reply client/src/components/HelloWorld.vue:1]: this as in HelloWorld.vue

File: client/src/components/fortune.vue (hunk 5 above)
Comment (GiantRobato) [reply client/src/components/fortune.vue:18]: The bootstrap issues says to do this to fix the column widths: https://github.com/twbs/bootstrap/issues/863

File: client/src/components/HelloWorld.vue (hunk 3 above)
Comment (GiantRobato) [reply client/src/components/HelloWorld.vue:1]: I'm using single file components and I'm pretty sure this is needed to differentiate between the html/script/css. https://vuejs.org/v2/guide/single-file-components.html

File: client/build/build.js (hunk 1 above)
Comment (GiantRobato) [reply client/build/build.js:1]: I don't think is a build folder but a folder that holds the build scripts... deleting it actually broke `npm run dev`

File: client/src/components/fortune.vue (hunk 4 above)
Comment (GiantRobato) [reply client/src/components/fortune.vue:6]: I'll move this to the css section.

File: client/src/components/fortune.vue (hunk 5 above)
Comment (GiantRobato) [reply client/src/components/fortune.vue:18]: I used css selectors so now it's down with the other css.

File: client/build/build.js (hunk 1 above)
Comment (ericpak) [reply client/build/build.js:1]: don't delete it from your machine. Just remove it from source control and add the build folder to the gitignore. The compiled code/minified code doesn't need to be stored in source control.

File: client/src/components/HelloWorld.vue (hunk 3 above)
Comment (ericpak) [reply client/src/components/HelloWorld.vue:1]: I don't see why it's needed. I don't see the component being used anywhere. and if it is it probably shouldn't be. But I'll take a closer look when i get home.

**Pull request merged**